### DISCUSSION TITLE
Chromecast from Android apk?

### DISCUSSION THREAD

**Discussion by as400jockey** [GerryDazoo/SlingerPlayer#13] [Ideas]
I was able to get VLC to cast from SlingerServer to a chromecast.  But had to set up a -750ms delay to get the audio to sync.  And it's a pita to get it started. And then there's the 5+ second lag between remote buttons and screen reactions.

Your android APK is BRILLIANT.  Simple.  FAST (only 1 second lag - hardly noticeable).  I LOVE that it's full screen held horizontally, and to get to the remote I turn the phone or tablet vertical.  Beep bopp boop, and back horizontal to full screen.

CASTing would be very handy.

Once again, a thousand thanks for all of this.

**Comment by GerryDazoo** [comment]
You can use a seperate app to cast whatever is on your screen.
https://www.cnet.com/tech/home-entertainment/2-easy-ways-to-cast-your-android-screen-to-your-tv/

**Reply by as400jockey** [reply]
thanks for that.  will give it a try.

**Reply by as400jockey** [reply]
Tested mirroring the device to the TV using Chromecast.  It works, but it's laggy and choppy, as often happens when mirroring the entire device.

the VLC approach of using the Chromecast 'renderer' works great in terms of scan rate, etc, but VLC is a pita with the audio delay, the 5 second remote button response, etc.

It would be a 'nice to have' to be able to Chromecast directly, but there are other ways to accomplish what I'm looking for.  My goal is to get rid of the $10/mo cable box that only gets used a handful of times each year (it's outside) for baseball or football games.  I think I'll use an old 'convertible' laptop with touch screen and connect directly with hdmi.  The "rotate 90 degrees to get the remote" feature will be great.
